src/ResourceManager/Sql/Commands.Sql/help/New-AzureRmSqlDatabase.md

```diff
@@ -1,3 +1,478 @@
+---

```

Comment:
@ahamad-MS please revert the changes made to this file and the `AzureRM.psm1` file below this one

Reply:
@cormacpayne  Fixed